jscs/auto-fix.js

```diff
@@ -0,0 +1,14 @@
+// Add support for a --noFix argument to disable JSCS auto-fix.
+
+module.exports = jscsAutoFix;
+
+function jscsAutoFix() {
+  var args = process.argv.slice(2);
+  var noFixArgIndex = args.indexOf('--noFix');
+  var noFix = noFixArgIndex > -1;
+
+  if (noFix) {
+    process.argv.splice(noFixArgIndex + 2, 1);
```

Comment:
@gzurbach once an options parser is introduced to this data flow we can take advantage of it in useful ways to avoid the need for modifying `process.argv` at all. e.g. [my other comment](https://github.com/godaddy/javascript/pull/28/files/12a8982f29f098d4ab581eda49aaaf16a07dd0fc#r58423483)

``` js 
 targets: process.argv.splice(2),
```

simply becomes

``` js
  targets: argv._
```